site/src/App.js

```diff
@@ -12,14 +12,40 @@ import Footer from './component/Footer/Footer';
 
 import SearchPage from "./pages/SearchPage";
 import CoursePage from "./pages/CoursePage/CoursePage.jsx";
+import ProfessorPage from './pages/ProfessorPage';
 import ErrorPage from "./pages/ErrorPage";
 import RoadmapPage from "./pages/RoadmapPage";
+
 import ZotisticsPage from "./pages/ZotisticsPage";
 import Schedule from "./component/Schedule/Schedule";
 import Sidebar from "./component/SideBar/SideBar";
 
+//Create a Client
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { gql } from '@apollo/client';
+import { ApolloProvider } from '@apollo/client/react';
+
+const client = new ApolloClient({
+    uri: 'http://localhost:5000/graphql/',
```

Comment:
Shouldn't hard code localhost:5000 to refer to server (would only work during development). I think /graphql would suffice.